docs/UserGuide.md

```diff
@@ -454,22 +456,32 @@ Demonstration:
 Adds a time period where the staff is working to the staff’s schedule.
 
 * There are two ways to identify the staff to add the time period to: by their `name` or by their staff `index`.
-* The `fulldayname` field required to specify shifts are not case sensitive.
 * The start time and end time will be set to the default one (If it's a morning slot, then the period of shift is from
   10:00 to 16:00; If it's an afternoon slot, then the period of shift is 16:00 to 22:00).
-* If no date input is provided, current week is taken as default.
-* If only one date input is provided, the shift added is to the next date that the shift is at. For instance, 
-  if the shift is on a 1/10/2021, a monday, with `da/2021-10-27` as input, the shift will be added to 1/10/2021.
+* If no `da/` input is provided, current week is taken as default.
+* If only one date input is provided, this input will be regarded as the start date bound the shift added is
+  to the next date that the shift is at. For instance, if the shift is on a 1/11/2021, a monday,
+  with `da/2021-10-27` as input, the shift will be added to 1/10/2021 (Because 1/11/2021 is the first
+  monday after 2021/10/27).
+* If two dates are provided, they will be considered as the start date and the end date of the period. Shifts will be
+  added to Any date matched with the `d/` input. For example, if the command
+  `addShift -n Candice d/Monday-1 da/2021-10-27 da/2021-11-14` is executed, shifts will be added to the afternoon of
+  1/11/2021, 8/11/2021, because these are the two mondays in this period.
 * If the shift is not in the period provided, the program will do nothing.
 
+
 Formats:  
-`addShift -n name d/fullDayName-shiftNumber [da/START_DATE] [da/END_DATE]`  
-`addShift -i index d/fullDayName-shiftNumber [da/START_DATE] [da/END_DATE]`
+`addShift -n NAME d/FULLDAYNAME-SHIFTNUMBER [da/START_DATE] [da/END_DATE]`
```

Comment:
I think we should specify what FULLDAYNAME refers to, because the name isn't very self explanatory. maybe in the explanation above? also if you're using fulldayname in this function, the other functions we have that uses the FULLDAYNAME should also be updated for consistency